x/bech32ibc/types/types.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	fmt "fmt"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
```

Comment:
Add comment saying this was copied from bech32 library
(find link to code)

Reply:
Done